update/Pos/Update/Constants.hs

```diff
@@ -82,6 +88,20 @@ curSoftwareVersion :: SoftwareVersion
 curSoftwareVersion = SoftwareVersion ourAppName
                                      (ccApplicationVersion updateConstants)
 
+ourSystemTag :: SystemTag
+ourSystemTag = $(do
+    mbTag <- runIO (lookupEnv "CSL_SYSTEM_TAG")
```

Comment:
Could we here just take different tags per OS instead of using bash env? It's easier for now as we don't really need to change this.

Reply:
Determining os if harder then just setting it from build script I guess